index.js

```diff
@@ -1,10 +1,44 @@
 const babel = require('babel-core');
+const fs = require('fs');
+const path = require('path');
 
-const babelOpts = {
-  plugins: ['transform-flow-strip-types', 'syntax-jsx']
+/**
+ * Remove duplicates from array
+ * @param  {array} items Input array
+ * @return {array} Output array with unique values
+ */
+const arrayUnique = (items) =>
```

Comment:
Thanks, I did consider this. I see the package already expects node > 4.4.x. Switched to this method.